static/app/components/preprod/preprodBuildsTable.tsx

```diff
@@ -39,12 +40,16 @@ export function PreprodBuildsTable({
   projectSlug,
   hasSearchQuery,
 }: PreprodBuildsTableProps) {
+  const labels = useMemo(
+    () => getLabels(builds[0]?.app_info?.platform ?? undefined),
```

Comment:
nit: I guess this depends on the user not uploading different platforms to the same project? maybe that's an edge case.